lib/ox/element.rb

```diff
@@ -150,7 +150,18 @@ def locate(path)
     # _raise_ [NoMethodError] if no match is found
     def method_missing(id, *args, &block)
       has_some = false
-      ids = id.to_s
+      ids = id.to_s.sub('_', ':') # allows easy API with namespaces
```

Comment:
I'm parsing something like this:

```
<entry xmlns="http://www.w3.org/2005/Atom" xmlns:gd="http://schemas.google.com/g/2005">
  <gd:name>
    <gd:givenName>First</gd:givenName>
    <gd:familyName>Last</gd:familyName>
    <gd:fullName>First Last</gd:fullName>
  </gd:name>
  <!-- ... -->
  <gd:structuredPostalAddress rel="http://schemas.google.com/g/2005#work" primary="true">
    <gd:city>Mountain View</gd:city>
    <gd:street>1600 Amphitheatre Pkwy</gd:street>
    <gd:region>CA</gd:region>
    <gd:postcode>94043</gd:postcode>
    <gd:country>United States</gd:country>
    <gd:formattedAddress>1600 Amphitheatre Pkwy Mountain View, CA 94043 United States</gd:formattedAddress>
  </gd:structuredPostalAddress>
  <!-- ... -->
</entry>
```

Without the substitution hack, I didn't know how to address `entry.gd_name.gd_givenName` (because `entry.gd:name.gd:givenName` can't work, right?). Perhaps something like `entry.gd.name.gd.givenName` could be done but I couldn't figure out how to do that without introducing pseudo-nodes into the parsed document.